src/products/components/ProductListPage/filters.ts

```diff
@@ -254,7 +255,11 @@ export function createFilterStructure(
         attr.slug,
         attr.name,
         Array.isArray(attr.value)
-          ? undefined
+          ? ["true", "false"].includes(attr.value[0])
```

Comment:
**question**: can it be extracted to its own function with more verbose ifs?

I'm asking as multiple ternaries are getting a bit unreadable